intrastat_base/tests/test_all.py

```diff
@@ -34,46 +28,3 @@ def test_no_email(self):
     def test_accessory(self):
         with self.assertRaises(ValidationError):
             self.shipping_cost.type = "consu"
-
-    def test_declaration_no_country(self):
```

Comment:
@alexis-via You should move those tests to intrastat_product

Reply:
done